VDP/game.py

```diff
@@ -0,0 +1,150 @@
+import sqlite3
+import os
+from random import choice, randrange
+from flask import request
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.exc import IntegrityError, OperationalError
+
+from VDP import app, APP_ROOT, db, logger
+import VDP.models
+from VDP.config_file import SQLALCHEMY_DATABASE_URI, SQLALCHEMY_TRACK_MODIFICATIONS
```

Comment:
je v poriadku takto rozdelit import na viac riadkov, ak je prilis vela znakov v riadku podla PEP8?

Reply:
Určitě můžeš na více řádků. :slightly_smiling_face: 
Buď na konec řádku dáš myslím zpětné lomítko a nebo (což mi přijde jako lepší způsob) dáš importované věci do závorek.
```python
from VDP.config_file import (SQLALCHEMY_DATABASE_URI, 
    SQLALCHEMY_TRACK_MODIFICATIONS)
```

Reply:
dakujem :)